### ISSUE TITLE
Скласти інтро

### ISSUE THREAD

**Issue by githubMazur** [githubMazur/homepage#8]
Скласти інтро (вступ), де в 2-4 реченнях описати, чим ти займаєшся, що подобається робити, що хочеться робити краще, що важливо робити, чому ти тут.

**Comment by githubMazur** [comment]
Hello, everyone! My name is Oleh. I am a junior Front-End Developer with a Bachelor’s degree in Computer Science. Over the past year, I have gained valuable experience as a React Native Developer, honing my skills and knowledge in mobile app development. I have completed several courses on Udemy and continue my journey of studying and self-improvement. I am highly motivated to apply my expertise to real-life projects, aiming to contribute effectively and expand my proficiency.

**Comment by githubMazur** [comment]
Hello, everyone! My name is Oleh. I am a junior Front-End Developer with a Bachelor’s degree in Computer Science. Over the past year, I have gained valuable experience as a React Native Developer, honing my skills and knowledge in mobile app development. I have completed several courses on Udemy and continue my journey of studying and self-improvement. I am highly motivated to apply my expertise to real-life projects, aiming to contribute effectively and expand my proficiency.
